configs/setups/awicm3/awicm3.yaml

```diff
@@ -724,6 +896,12 @@ oasis3mct:
                                 rstos.nc:
                                         - 'A_SST:A_Ice_frac:A_Snow_thickness:A_Ice_temp:A_Ice_albedo <--gauswgt_i-- sst_feom:sie_feom:snt_feom:ist_feom:sia_feom'
 
+        # TODO: change this to a control on the patch version general.version
```

Comment:
Same as above, we have to discuss how we want to control this.

Reply:
lets have a chat next week.